pinot-common/src/main/antlr4/org/apache/pinot/pql/parsers/PQL2.g4

```diff
@@ -78,6 +78,7 @@ predicate:
   | isClause                              # IsPredicate
   | regexpLikeClause                      # RegexpLikePredicate
   | textMatchClause                       # TextMatchPredicate
+  | likeClause                            # TextMatchPredicate
```

Comment:
Why are we changing PQL grammar ? PQL endpoint has been deprecated and will be removed in future